### ISSUE TITLE
Retrieve capability infos from the cluster instead of hardcoding data

### ISSUE THREAD

**Issue by metacosm** [halkyonio/hal#69]
`hal capability create` should retrieve the live data from the cluster now that it's possible instead of hardcoding values.

**Comment by cmoulliard** [comment]
List of versions is empty @metacosm 
```
oc get postgresversions.catalog.kubedb.com
NAME       VERSION   DB_IMAGE                   DEPRECATED   AGE
10.2       10.2      kubedb/postgres:10.2       true         35h
10.2-v1    10.2      kubedb/postgres:10.2-v2    true         35h
10.2-v2    10.2      kubedb/postgres:10.2-v3                 35h
10.2-v3    10.2      kubedb/postgres:10.2-v4                 35h
10.2-v4    10.2      kubedb/postgres:10.2-v5                 35h
10.6       10.6      kubedb/postgres:10.6                    35h
10.6-v1    10.6      kubedb/postgres:10.6-v1                 35h
10.6-v2    10.6      kubedb/postgres:10.6-v2                 35h
11.1       11.1      kubedb/postgres:11.1                    35h
11.1-v1    11.1      kubedb/postgres:11.1-v1                 35h
11.1-v2    11.1      kubedb/postgres:11.1-v2                 35h
11.2       11.2      kubedb/postgres:11.2                    35h
9.6        9.6       kubedb/postgres:9.6        true         35h
9.6-v1     9.6       kubedb/postgres:9.6-v2     true         35h
9.6-v2     9.6       kubedb/postgres:9.6-v3                  35h
9.6-v3     9.6       kubedb/postgres:9.6-v4                  35h
9.6-v4     9.6       kubedb/postgres:9.6-v5                  35h
9.6.7      9.6.7     kubedb/postgres:9.6.7      true         35h
9.6.7-v1   9.6.7     kubedb/postgres:9.6.7-v2   true         35h
9.6.7-v2   9.6.7     kubedb/postgres:9.6.7-v3                35h
9.6.7-v3   9.6.7     kubedb/postgres:9.6.7-v4                35h
9.6.7-v4   9.6.7     kubedb/postgres:9.6.7-v5                35h

hal capability create                  
? Category database
? Type Postgres
? Version  [Use arrows to move, space to select, type to filter]
❯ 
```
But versions defined under`capabilityinfo` for postgresql is empty
<img width="1382" alt="Screenshot 2020-01-23 06 53 33" src="https://user-images.githubusercontent.com/463790/72960200-32415380-3dad-11ea-8649-4995ca812107.png">

**Comment by metacosm** [comment]
Something is going on with your cluster and the plugin… Unfortunately, this is going to be tricky to debug…

**Comment by metacosm** [comment]
Fixed by halkyonio/operator#206
